package.json

```diff
@@ -59,7 +59,7 @@
     "sscce-mysql": "cross-env DIALECT=mysql yarn sscce",
```

Comment:
I can't comment on line 21, but that should be updated as well

Reply:
Done